password_security/models/res_users.py

```diff
@@ -81,6 +81,8 @@ def check_password(self, password):
             password_regex.append(r'(?=.*?\W)')
         password_regex.append('.{%d,}$' % company_id.password_length)
         if not re.search(''.join(password_regex), password):
+            self.env.cr.rollback()
```

Comment:
Yeah, you are right!
I deleted the rollback and it's working just using invalidate_cache

For a strange reason the website frontend is not cleaning the cache.
Because the same exception using backend is working without clean cache.

So, maybe this is a issue related with website but I didn't find a way to reproduce it without a custom module and will be hardly that odoo fixed.